server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistComponentsStep.java

```diff
@@ -56,12 +61,16 @@
   private final TreeRootHolder treeRootHolder;
   private final MutableDbIdsRepository dbIdsRepository;
   private final System2 system2;
+  private final MutableDisabledComponentsHolder disabledComponentsHolder;
 
-  public PersistComponentsStep(DbClient dbClient, TreeRootHolder treeRootHolder, MutableDbIdsRepository dbIdsRepository, System2 system2) {
+  public PersistComponentsStep(DbClient dbClient, TreeRootHolder treeRootHolder,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)